### ISSUE TITLE
Theme app extension missing from 2.0 themes

### ISSUE THREAD

**Issue by robinsharma** [Shopify/dawn#485]
**Describe the current behavior**
_TLDR; Published theme app extension not showing up on 2.0 themes_

We recently published our theme app extension for our Shopify app. Everything is working as intended on our Dawn development themes. However, we have merchants with our app along with new merchants signing up using online store 2.0 themes, other than Dawn, that don't see any of our app blocks or app embed blocks.

**Describe the expected behavior**
App blocks and app embed blocks should show on all 2.0 themes

**Version information (Dawn, browsers and operating systems)**
- Dawn Version: 1.0.0
- Chrome Version 91.0.4472.114
- macOS Version 11.3.1

**Possible solution**

**Additional context/screenshots**
Dawn (development):
<img width="1552" alt="Screen Shot 2021-08-25 at 1 59 01 PM" src="https://user-images.githubusercontent.com/4361226/130842349-befa4c1a-30be-42e9-ab1a-883c84b27009.png">

Merchant with our app running a different online store 2.0 theme:
<img width="1552" alt="Screen Shot 2021-08-25 at 1 58 16 PM" src="https://user-images.githubusercontent.com/4361226/130842420-dd025148-1cd9-4118-b0ef-d7f800526718.png">
